### PR TITLE
Avatar

### PR DESCRIPTION
**Pull request by rdthetha**

### CODE REVIEW

File: components/avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    width: 15rem;
+    height: 15rem;
+}
+
+.avatar-text {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    color: aliceblue;
+    background-color: #1c0221;
```
Comment (anasm11) [inline components/avatar/avatar.css:34]: better to give in variables. rest all looks good

Comment (rdthetha) [reply components/avatar/avatar.css:34]: thank you! fixed it

**Comment by amantulsyan35** [comment]
looks good to merge 👍

**Comment by amantulsyan35** [comment]
comments in CSS files and at index.html would increase the readability more

File: components/avatar/avatar.html [hunk 2]
```diff
@@ -22,31 +22,57 @@
...
-                on the grid.</p>
-        </article>
+        <section class="main-section">
+            <div class="section-heading">Avatar</div>
+            <div class="section-subhead">Avatar has many use cases of different sizes and types for eg.profile pictures
+            </div>
+            <p class="section-para">In the below example we have used image in an avatar form. Use classes avatar-xs
+                ,avatar-sm,avatar-md and avatar-lg .</p>
+            <div class="component-example flex-row flex-align-center flex-gap1 first">
+                <div><img class="avatar avatar-xs" src="face.JPG" alt=""></div>
```
Comment (Ankur9669) [inline components/avatar/avatar.html:51]: alt tags should not be empty and also the div could be removed here...

File: components/avatar/avatar.html [hunk 3]
```diff
@@ -22,31 +22,57 @@
...
+                <div><img class="avatar avatar-md" src="face.JPG" alt=""></div>
+                <div><img class="avatar avatar-lg" src="face.JPG" alt=""></div>
+            </div>
+            <iframe
+                src="https://carbon.now.sh/embed?bg=rgba%28200%2C173%2C172%2C1%29&t=panda-syntax&wt=none&l=css&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%253E%250A%253Cimg%2520class%253D%2522avatar%2520avatar-xs%2522src%253D%2522face.JPG%2522%2520alt%253D%2522%2522%253E%253C%252Fdiv%253E%250A%253Cdiv%253E%250A%253Cimg%2520class%253D%2522avatar%2520avatar-sm%2522src%253D%2522face.JPG%2522%2520alt%253D%2522%2522%253E%253C%252Fdiv%253E%250A%253Cdiv%253E%250A%253Cimg%2520class%253D%2522avatar%2520avatar-md%2522src%253D%2522face.JPG%2522%2520alt%253D%2522%2522%253E%253C%252Fdiv%253E%250A%253Cdiv%253E%250A%253Cimg%2520class%253D%2522avatar%2520avatar-lg%2522src%253D%2522face.JPG%2522%2520alt%253D%2522%2522%253E%253C%252Fdiv%253E"
+                style="width: 614px; height: 318px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (Ankur9669) [inline components/avatar/avatar.html:58]: Remove inline styling which comes from carbon.
Create one css class for this.

File: components/avatar/avatar.css [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+.avatar-lg {
+    width: 15rem;
+    height: 15rem;
+}
+
+.avatar-text {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    color: aliceblue;
```
Comment (Ankur9669) [inline components/avatar/avatar.css:33]: You can use css variable here

**Review by Ankur9669 — CHANGES_REQUESTED** [review]
Great Code!!!. I loved the name of you site. The naming conventions are also awesome.

File: components/avatar/avatar.html (hunk 2 above)
Comment (rdthetha) [reply components/avatar/avatar.html:51]: sure would fix it

**Pull request merged**